firebase-vscode/package.json

```diff
@@ -133,5 +133,8 @@
     "webpack": "^5.75.0",
     "webpack-cli": "^5.0.1",
     "webpack-merge": "^5.8.0"
-  }
+  },
```

Comment:
Suspect these slipped in due to local vscode dev, probably should not be included here.